be/src/vec/data_types/serde/data_type_bitmap_serde.cpp

```diff
@@ -81,12 +82,12 @@ Status DataTypeBitMapSerDe::read_column_from_pb(IColumn& column, const PValues&
 
 void DataTypeBitMapSerDe::write_one_cell_to_jsonb(const IColumn& column, JsonbWriter& result,
                                                   Arena* mem_pool, int32_t col_id,
-                                                  int row_num) const {
+                                                  int64_t row_num) const {
     auto& data_column = assert_cast<const ColumnBitmap&>(column);
-    result.writeKey(col_id);
+    result.writeKey(cast_set<JsonbKeyValue::keyid_type>(col_id));
     auto bitmap_value = const_cast<BitmapValue&>(data_column.get_element(row_num));
     // serialize the content of string
-    auto size = bitmap_value.getSizeInBytes();
+    auto size = cast_set<uint32_t>(bitmap_value.getSizeInBytes());
```

Comment:
why here need cast  ? if the byte over uint32_t is a danger behaviror

Reply:
done